tasks/configure/tagged_volume.yml

```diff
@@ -28,7 +28,7 @@
   when: tagged_volume_mount_facts.volumes[0] is not defined
   until: tagged_volume_facts.volumes[0].id is defined
   retries: 6
```

Comment:
You don't want to control retries here? :)

Reply:
missed that one, adding